rootfs/usr/share/inputplumber/devices/50-ayaneo_2021.yaml

```
@@ -48,9 +48,10 @@ source_devices:
     iio:
       name: i2c-10EC5280:00
       mount_matrix:
+        # TODO: Reverify on actual hardware
         x: [0, -1, 0]
-        y: [0, 0, -1]
-        z: [1, 0, 0]
+        y: [1, 0, 0]
```

Based on your diagram of positive/negative deflection, this should be inverted.
```suggestion
        y: [-1, 0, 0]
```